.github/workflows/esti.yaml

```diff
@@ -868,6 +868,15 @@ jobs:
     name: Build metadata client for Spark 3.x
     runs-on: ubuntu-20.04
     needs: check-secrets
+    strategy:
+      matrix:
+        spark:
+          - project-variable: core3
+            project-suffix: 301
+            artifact-suffix: ""
```

Comment:
Yes - we need to somehow give these two artifacts different names.
I agree it could be simpler, I will organize it when we revisit our Spark support matrix.